modules/EnsEMBL/Web/Component/Location/SyntenyImage.pm

```diff
@@ -151,7 +151,7 @@ sub species_form {
   my @values;
 
   foreach my $next (@sorted) {
-    next if $next->{'name'} eq $hub->species;
+    next if $next->{'name'} eq $hub->species and $next->{'name'} ne 'Homo_sapiens'; # Homo sapiens is the only vertebrate that has self-synteny
```

Comment:
I'm not sure if Perl would correctly intepret the combination of `and` and `or` in that line. I think it would be good to enclose the logic after the "if" in round brackets, in order to avoid the possibility of Perl misinterpreting the priority/binding of the "and". i.e. make the line
`next if ( $next->{'name'} eq $hub->species and $next->{'name'} ne 'Homo_sapiens' );`